src/site/_data/postToPaths.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const paths = require('./paths/all.js');
+
+const postToPathsMap = {};
+Object.keys(paths).forEach((path) => {
+  (paths[path].topics || []).forEach((topic) => {
+    (topic.pathItems || []).forEach((slug) => {
```

Comment:
Note we also support subtopics, which the progressive-web-apps path uses.
https://github.com/GoogleChrome/web.dev/blob/master/src/site/content/en/progressive-web-apps/progressive-web-apps.11tydata.js#L35